File: eternal_return/src/main/java/org/EternalReturn/ERCharacter/Character/isaac/Character_Isaac.java
package org.EternalReturn.ERCharacter.Character.isaac;

import org.EternalReturn.ERCharacter.Character.isaac.Active;
import org.EternalReturn.ERCharacter.Character.isaac.Passive;
import org.EternalReturn.ERCharacter.ERCharacter;
import org.EternalReturn.ERPlayer.ERPlayer;

public class Character_Isaac extends ERCharacter {

    public boolean isActiveSkill = false;

    public Character_Isaac(ERPlayer player) {
        super(player);
        this.cooldownSeconds = 8;
        registerMonobehaviour(new Passive());
        registerMonobehaviour(new Active());
    }


    @Override
    public String getName() { return "isaac"; }

}

File: eternal_return/src/main/kotlin/org/eternalreturn/ercharacter/character/isaac/Character_Isaac.kt
package org.eternalreturn.ercharacter.character.isaac

import org.bukkit.entity.Player
import org.eternalreturn.ercharacter.datastructure.CoolTableSeconds
import org.eternalreturn.erplayer.DPlayer
import org.eternalreturn.system.EREngine

class Character_Isaac(erEngine : EREngine, player: Player) : DPlayer(player, erEngine) {
    var isActiveSkill: Boolean = false
    override val activeCoolForEachLevel  = CoolTableSeconds(this::activeLevel, 60, 60, 60, 60, 60)
    override val passiveCoolForEachLevel = CoolTableSeconds(this::passiveLevel,0, 0, 0, 0, 0)

    init {
        this.ActiveCooldownSeconds = 3
        this.PassiveCooldownSeconds = 8
        registerMonobehaviour(Passive())
        registerMonobehaviour(Active())
    }


    override val name: String
        get() = "isaac"
}